### PR TITLE
Added top entity lookup files

### PR DESCRIPTION
**Pull request by psykokwak4**
- The replacement for relative semantics in include statements depends on CMake change, will change it in future commits.
- Moved generated code to `sdk/cpp/ydk/tests/models_gen`
- The ydk-gen before this pull request uses module name as namespace

### CODE REVIEW

File: ydkgen/printer/cpp/entity_lookup_printer.py [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+            packages = sorted(packages, key=lambda p: p.name)
+
+        self._init_headers(packages)
+        self._init_insert_stmts(packages, sort_clazz)
+
+        self._print_headers()
+        self._print_get_entity_lookup_func()
+
+    def _init_headers(self, packages):
+        unique_headers = set()
```
Comment (ghost) [inline ydkgen/printer/cpp/entity_lookup_printer.py:42]: Can this function be split into smaller functions per the clean code principles?

File: ydkgen/printer/cpp/entity_lookup_printer.py [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+                entity_lookup[path] = top_entity.fully_qualified_cpp_name()
+
+        self.entity_lookup = entity_lookup
+
+    def _print_headers(self):
+        for header in self.headers:
+            self.ctx.writeln(header)
+
+    def _print_get_entity_lookup_func(self):
+        self.ctx.bline()
```
Comment (ghost) [inline ydkgen/printer/cpp/entity_lookup_printer.py:90]: Can this function be split into smaller functions per the clean code principles?

File: ydkgen/printer/cpp/entity_lookup_printer.py (hunk 1 above)
Comment (psykokwak4) [reply ydkgen/printer/cpp/entity_lookup_printer.py:42]: Sorry, will change in future commits

File: ydkgen/printer/cpp/entity_lookup_printer.py (hunk 2 above)
Comment (psykokwak4) [reply ydkgen/printer/cpp/entity_lookup_printer.py:90]: Sorry, will change this in future commits

**Pull request merged**